### ISSUE TITLE
:deobfMcMCP during launchClient build fails

### ISSUE THREAD

**Issue by Zootar** [Microsoft/malmo#172]
:deobfMcMCP FAILED during running launchClient.sh. lauchClient.sh runed with --stacktrace and --debug (log attached).
OS - RHEL 7,kernel 3.10.0-327.22.2.el7.x86_64.

[launchClient.txt](https://github.com/Microsoft/malmo/files/359005/launchClient.txt)

**Comment by Zootar** [comment]
Issue already solved

**Comment by DaveyBiggers** [comment]
@Zootar can you put a link to, or a description of, the solution? That'll help anyone else who is having the same issue.

**Comment by Zootar** [comment]
Probably there was problem with some packages I supposed that was java-1.8.0-openjdk-devel.
I tried firsly install all dependencies to just dependencies from standard installation dependencies:
sudo yum install boost python java-1.8.0-openjdk xerces-c ffmpeg mono lua lua-socket
but build failed with error from topic.

Next I installed all dependencies to build from source code:
sudo yum install git cmake cmake-gui boost-devel python-devel java-1.8.0-openjdk-devel swig xsd xerces-c-devel doxygen libxslt ffmpeg gcc-c++ mono-devel lua lua-devel lua-socket

I successfully build from source so I tried do again install simplified version and it passed successfully. Some of these packages below repaired problem.
